arch/src/template/mod.rs

```diff
@@ -0,0 +1,30 @@
+use memory_model::{GuestAddress, GuestMemory};
```

Comment:
As mentioned before, I think this should be turned into a stub for `aarch64`, instead of a generic template. If it will eventually turn into `aarch64` we might as well do it now. Otherwise, I don't really like something like this lingering in the repository. If its purpose was to document the required functionality that ppl must implement to support new architectures, we could move all this information in an actual doc somewhere.

Reply:
See grand-previous comment.